vignettes/epipredict.Rmd

```diff
@@ -1,470 +1,565 @@
-## Why doesn't this package already exist?
-
-As described above:
+This dataset uses a single key, `geo_value`, and two separate
+time series, `case_rate` and `death_rate`.
+The keys are represented in "long" format, with separate columns for the key and
+the value, while separate time series are represented in "wide" format with each
+time series stored in a separate column.
 
-* Parts actually DO exist. There's a universe called `{tidymodels}`. It handles
-preprocessing, training, and prediction, bound together, through a package called
-`{workflows}`. We built `{epipredict}` on top of that setup. In this way, you CAN
-use almost everything they provide.
+`{epiprocess}` is designed to handle data that always has a geographic key, and
+potentially other key values, such as age, ethnicity, or other demographic
+information.
+For example, `grad_employ_subset` from `{epidatasets}` also has both `age_group`
+and `edu_qual` as additional keys:
 
-* However, `{workflows}` doesn't do postprocessing. And nothing in the -verse
-handles _panel data_.
-
-* The tidy-team doesn't have plans to do either of these things. (We checked).
-
-* There are two packages that do _time series_ built on `{tidymodels}`, but it's
-"basic" time series: 1-step AR models, exponential smoothing, STL decomposition,
-etc.[^2] Our group has not prioritized these sorts of models for epidemic
-forecasting, but one could also integrate these methods into our framework.
+```{r extra_keys}
+grad_employ_subset
+```
 
-[^2]: These are [`{timetk}`](https://business-science.github.io/timetk/index.html)
-and [`{modeltime}`](https://business-science.github.io/timetk/index.html). There
-are *lots* of useful methods there than can be used to do fairly complex machine
-learning methodology, though not directly for panel data and not for direct
-prediction of future targets.
+See `{epiprocess}` for [more details on the `epi_df` format](https://cmu-delphi.github.io/epiprocess/articles/epi_df.html).
 
-# Show me the basics
+Panel time series are ubiquitous in epidemiology, but are also common in
+economics, psychology, sociology, and many other areas.
+While this package was designed with epidemiology in mind, many of the
+techniques are more broadly applicable.
 
-We start with the `jhu` data displayed above. One of the "canned" forecasters we
-provide is an autoregressive forecaster with (or without) covariates that
-*directly* trains on the response. This is in contrast to a typical "iterative"
-AR model that trains to predict one-step-ahead, and then plugs in the
-predictions to "leverage up" to longer horizons.
+## Customizing `arx_forecaster()`
+Let's expand on the basic example presented on the [landing
+page](../index.html#motivating-example), starting with adjusting some parameters in
+`arx_forecaster()`.
 
-We'll estimate the model jointly across all locations using only the most
-recent 30 days.
+The `trainer` argument allows us to set the fitting engine. We can use either one of the
```

Comment:
I don't like "fitting". Here, I would say either "computational" or "estimation"
(neither is quite correct).